halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/ui/v1/Ui.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.cli.ui.v1;
+
+/**
+ * A shared colored terminal output class
+ */
+public class Ui {
+  private Color color;
+  private boolean verbose;
+
+  public Ui(boolean colorEnabled, boolean verbose) {
+    this.color = new Color(colorEnabled);
+    this.verbose = verbose;
+  }
+
+  private void reset(String message) {
+    try {
+      System.out.print(message);
+    } catch (Exception e) {
+      throw e;
+    } finally {
+      System.out.println(color.RESET);
+    }
+  }
+
+  public void raw(String message) {
+    reset(message);
+  }
+
+  public void info(String message) {
+    if (verbose) {
+      reset(color.BOLD + color.BLUE + ". " + color.RESET + message);
+    }
+  }
+
+  public void warning(String message) {
```

Comment:
Last remark. This is specific for CLI (ok since it is in the CLI package). For the above suggestion you want an API where the "flavor" is injected based on the output media you are rendering for (i.e. BOLD, YELLOW are abstract interface, not static constants), so a Web UI would define the color and weight differently to give you the right CSS. Then the call sites could have multi-color text that renders properly in browser or shell. Maybe there are existing ways to do this.